src/user.rs

```diff
@@ -39,7 +39,7 @@ impl User {
 
 /// Crate-internal representation of a Zitadel user
 #[derive(Clone, Debug)]
-pub struct ZidatelUser {
+pub struct ZitadelUser {
```

Comment:
Let's put this in a separate commit, this just muddies the blame.

Reply:
This is also already corrected on main

Reply:
Yep, spotted that after, ta @emgrav